components/script/dom/htmlmeterelement.rs

```diff
@@ -52,4 +57,138 @@ impl HTMLMeterElement {
 impl HTMLMeterElementMethods for HTMLMeterElement {
     // https://html.spec.whatwg.org/multipage/#dom-lfe-labels
     make_labels_getter!(Labels, labels_node_list);
+
+    /// <https://html.spec.whatwg.org/multipage/#concept-meter-actual>
+    fn Value(&self) -> Finite<f64> {
+        let min = *self.Min();
+        let max = *self.Max();
+
+        Finite::wrap(
+            self.upcast::<Element>()
+                .get_string_attribute(&local_name!("value"))
+                .parse_floating_point_number()
+                .map_or(0.0, |candidate_actual_value| {
+                    if candidate_actual_value < min || min > max {
+                        return min;
+                    }
```

Comment:
Why is this needed? Isn't clamp enough?

Reply:
Oh avoiding panics while working on WPT passes, I can't reproduce those cases now, so removed 👍